### PR TITLE
LPD-31415

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
ci:test:echo:playwright

**Comment by ealonso** [comment]
ci:test:echo-playwright

File: modules/test/playwright/tests/layout-admin-web/millerColumns.spec.ts [hunk 1]
```diff
@@ -129,6 +130,74 @@ test('checks the correct label for restricted pages in Miller Columns', async ({
...
+
+	// Can view the draft label of a page in pages admin
+
+	await expect(
+		page.locator('.miller-columns-item').getByText('Draft')
+	).toBeVisible();
+
+	// Can view preview draft action
+
+	await clickAndExpectToBeVisible({
```
Comment (veroglez) [inline modules/test/playwright/tests/layout-admin-web/millerColumns.spec.ts:172]: It looks like the test is failing [here](https://ff19457a59e4ba4cde2796f5b9258308277c89fadc6eb80737bb2ae-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-28%2Ftest-portal-acceptance-pullrequest(master)%2F9786%2Fplaywright-js-tomcat90-mysql57-jdk8%2F22%2F0%2Fplaywright-report%2Findex.html?jk=AYBlUPDQkdozOG3_5DyH2zHA7wJJKXA8_Sy8TU4zoyrK0efYNYUTTO7AXKZX_abdNf8v0fm88XYQUQDNl9J_n4mjaNsEfrADoNkMo5yu1i8RlApXV6VV_kr12VocEAE6CbVeREWIiv6W3N2FEH6aqpANBXK2EMgEpMqWClym3h-lIcnSsEQXd4u9xyjwLHTC5JW6EMOxcgYMa8t41BYjOYM2e59WUJArv5GXRGCZMTkxBkqHuLn1aalprOYonKODHWYYMmDOLNd-868UiHkS3sOSp5EzeHcZkiaWXqPx2scuXe1dET1atdQy7ZwCCSSf3ie6_9q-0weARDFwgxj7kP7F4DgDn_Q1b5j7pPcKkp-IoRxjlGJ15dsI7xu449UxynvOf1L39S84Ngf3L5YvrS830uwvWo8L4WYqVHEHnJw5WGsNP3YcK-CnuRcH57BFGXKAz01QfZtNobBOXDwywdwe81d1MbkZzYJTfNjg74lnnYkxj2Q16NVF9HQrW3OMeupTgHW_WllPXt9hhHAYK4-CLWQQuMv1fRouFaqToLH3WHkCzv_XUtJjyem3eJYkF1vPKzgHL7shaXG7XU8hOAuVVtt_V0jvE0dtCIVBBcjSDwKEMrrRgWNTF7PsOM3a8tsT__J6uxVh21HADt1f1b2xiNvgZ8zTWEwO2B5p1ssTkZV8qStS0B2-V8AKOcZ0VSKha2-Qo_wg1X9TN5cgVoVuBiy6I0UgdbF5rPN3ucKUjndx71dSt-uDCX1fYkZf_F9ovItQ8jIrZrXCXcVgM59E0QUhLP8gLnirryVm776o4mLl8bQXamkcLy0s0Klen6RcVkwzn0zbb82CVxWWsd5MqJw6Ji-GMtRDOiyjJfvj8VLgEvVOa92UWeFkvNekG84JUvCJYWZ16AKj4K09QORQZHMu7g1vb251JSCvet8tZhYfR8S7MgnSe1KQGc_AjSPNVxIL4ecBgAFsAE0wnYOMJR0dhHSN1H01Ka4ivsUauvD8z9j_1VSi5wAu_hm_aou28niWE4DKYph5bxZZvp89HU0cMQ4qgYaKheYZPNapX6KRqNlpO-oNfGXD99gLcDyY1VHF0QlZw4NITw8PW6XzFCxB_ccbhPzxojQv9wqHfa0_ov-6PWkcoWFlL8o2cmCPzlFoEzUJJLN51TIYI4CyhCSPB_JhIZW3SgpOxwYopvpjGAbebItV19Q4CGmeAjNZP72W8eJpYYo_6hY4PuAl-oG09yD9PWNfyNQq5b2Bl4j6I63hC9n4uhUuMt4bLxSvARXjvLcqbEwCm6eFUfeJoF2CdlgVNWuirE4WQ-ZkHv8xCw&isca=1#?testId=ebc7874d8d96667aad39-daaae573f786787a46d3). What exactly do you want to do? check that the "Preview Draft" option appears?

Comment (ealonso) [reply modules/test/playwright/tests/layout-admin-web/millerColumns.spec.ts:172]: Yes, I want to check if the dropdown action appears

File: modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts [hunk 2]
```diff
@@ -160,6 +198,10 @@ test('Can edit the page name and layout template via pages administration.', asy
 
 	await page.getByTitle('1 Column', {exact: true}).click();
 
+	await expect(
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts:201]: I refactored this part 😄

File: modules/test/playwright/tests/layout-admin-web/millerColumns.spec.ts (hunk 1 above)
Comment (sandrodw3) [reply modules/test/playwright/tests/layout-admin-web/millerColumns.spec.ts:172]: I fixed this

**Comment by sandrodw3** [comment]
https://github.com/liferay-echo/liferay-portal/pull/15968

**Pull request closed without merging**